src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -655,7 +672,7 @@ public string SelectedPackagePathForInstall
         /// <summary>
         /// Settings that apply to view extensions.
         /// </summary>
-        public List<ViewExtensionSettings> ViewExtensionSettings { get; set; }
+        public List<ViewExtensionSettings> ViewExtensionSettingsList { get; set; }
```

Comment:
Would this cause old settings to not deserialize correctly right?

Reply:
this is an API break.

Reply:
Ok will revert, but this make our xml look like this:

```
< ViewExtensionSettings >
< ViewExtensionSettings></ViewExtensionSettings>
< ViewExtensionSettings></ViewExtensionSettings>
</ViewExtensionSettings>
```

Parent and child nodes have the same name.

Reply:
if you want to make this change you should figure out how to handle the old deserialization and keep the old property working/obsolete it - you don't know who is using it.